packages/manager/modules/pci/src/projects/project/kubernetes/components/billing-type/billing-type.component.js

```diff
@@ -4,7 +4,7 @@ export default {
   bindings: {
     flavor: '<',
     number: '<',
-    monthlyBilling: '<',
+    monthlyBilling: '=',
```

Comment:
This component looks similar to https://github.com/ovh/manager/tree/master/packages/manager/modules/pci/src/components/project/flavor-billing. Maybe it could make sense to reuse

Reply:
In fact `billing-type` uses `flavor-type`. It just adds a description around the component. I will directly use the `flavor-type` component as you suggest